compiler/test/formatter_outputs/application_indenting.gr

```diff
@@ -0,0 +1,29 @@
+let flagsToWasmVal = (flag, i) => {
+  let riiiiiiightsInheriting = Module64.load(structPtr, 16n)
+  (
+    riiiiiiightsInheriting &
+    1N << Module64.extendI32U(Module32.fromGrain(i) >> 1n)
+  ) >
+  0N
+}
+
+let grisuRound = (buffer, len, delta, rest, ten_kappa, wp_w) => {
+  let mut lastp = buffer + len - 1n
+  let mut digit = Module32.load8U(lastp, 0n)
+  let mut rest = rest
+  while (
+    Module64.ltU(rest, wp_w) &&
+    Module64.geU(Module64.sub(delta, rest), ten_kappa) &&
+    (
```

Comment:
I don't _love_ this. Here's what Prettier did though: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAbOMAEBzATgSwGcBXAJQmKgBNMBeTACgCNiAzVuXAGk3Sh6pxUMAIY9ccQjB7woAfQDWIgA7KxmAO7K5GgJR0AfJmAAdKJl4ZeIqcrqYW7TpgDUl8wFpMARihmL6FhU+Nj4WPQAshBUxOgAzABMAHSoECJUABwAqgyoNjDKPAAMULr+llgSUvZVMOUaABb46IzlFlEx6ABsACwpMDm1PFo6+gBkY22YHbFwvUnYcDkz3X0kTAyCwuq1ujIIiipq45PmFq1n59PRs-PCyzerSelUDEOYsoeqInua2nqYAA+gKm7Uecz62AGDFB5xWEKS6wYIw04kkMD2sLBnQRSPh8xeb3R+3kSm+vxRZUu5ypV1pmH0pmpwVCWA89F8U1q9nxfUJ70+ZOO5QAvuV4YlETAIBIMrl8oVMCywsVSmYRSAuCAIMoYPhoIRkKARLhcBANAAFE0IQ0oESoDQiACehq1TFwIjACgwAGURABbOAAGXwUDgyFY9sIcDdHq9vrUYFD2GQMFwxBjIDg-qYcCogioQZEUGwxBEiwAYjL-SIYHqS8gQCJiNLNSAGjB-agAOpNeCERNwH02sL4ABuYSdjbAhFdIFD0dwMAtHuwNYjUczACtCAAPH3J9AARWIEHgG9Q0a1alwi8bTBEudQbeUBFg3fwVBgDWQGSK17NaNuw9ZRG1fSRODHcMtQAR1PeAVx1W0m0IDwwzzPM2wkOD8AkFdy3XJBI0vTNo39fBU3TUjDzgE8z3DIjNy1UQmA-L8fyQBJmI9ZpkwAYQgf1CKzQgAFY22IaMABVH1tYirxAMcMwASWoBAYB9MACF1ABBagfRgJ10AvaMRRFIA

Reply:
I've made a tweak, it now matches prettier, let me know what you think about the other changes this produces.   I think they look OK